AirCasting/Graph/Graph.swift

```diff
@@ -103,21 +105,42 @@ class AirCastingGraph: UIView {
         if !didMoveOrScaleGraph && isAutozoomEnabled {
             zoomoutToThirtyMinutes(dataSet: dataSet)
         }
+        callDateRangeChangeObserver()
     }
     
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    private func callDateRangeChangeObserver() {
+        let dateRange = getCurrentDateRange()
+        onDateRangeChange?(dateRange)
+    }
+    
+    private func getCurrentDateRange() -> (start: Date, end: Date) {
+        let startDate = Date(timeIntervalSince1970: lineChartView.lowestVisibleX)
+        let endDate = Date(timeIntervalSince1970: lineChartView.highestVisibleX)
+        // Workaround for a weird quirk with Chart - when first called
+        // `startDate` is sane, but `endDate` is 1970, so we need a special
+        // case to fix that.
```

Comment:
I'm like 50/50 about a warning here. The reason is that usualy a warning would be something that can be fixed by us (like a todo or smth). However n this case we're totally dependent on an external library version. It wolud be cool to be able to set a warning here for future versions of graph so if we update it we suddenly get this warning (or an error even) saying "hey, this library was updated so check if this workaround is still needed" - sadly afaik it's not possible with xcodebuild... But on the other hand maybe a permanent warning here is better than no warning at all 🤷‍♂️

After giving it some thought I came to a conclusion that yea - let's leave a mark so we don't forget (although I think it might get lost in a pool of other warnings anyway)

Reply:
I agree we have lots of warnings. I plan to move them to Trello as small bugs or sth but I'm leaving it as post-MVP work right now :D